sos/plugins/networking.py

```diff
@@ -154,6 +155,54 @@ def setup(self):
             pred=SoSPredicate(self, kmods=['ip6table_filter'])
         )
 
+        # Get devlink output
+        self.add_cmd_output([
+                    "devlink dev",
+                    "devlink dev param show",
+                    "devlink dev info",
+                    "devlink port",
+                    "devlink sb show",
+                    "devlink sb pool show",
+                    "devlink sb port pool show",
+                    "devlink sb tc bind show",
+                    "devlink -s -v trap show"
+        ])
+
+        # Collect information about devlink devices and dpipe tables
+        devlink_dev = self.exec_cmd("devlink dev")
+        if devlink_dev['status'] == 0:
+            devices = []
+            dev_tables = []
+            for line_dev in devlink_dev['output'].splitlines():
+                devices.append(line_dev)
+                # Collect information about devlink dpipe tables
+                devlink_dev_tables = self.exec_cmd(
+                    "devlink dpipe table show %s" % line_dev
+                    )
```

Comment:
Since we're calling this command later on on line 194, instead of `exec_cmd` we can use `collect_cmd_output` which will both save the output to the archive and give you the output back directly in the `output` dict key.

In this loop there's nothing I immediately see that would need to change, we'd just need to remove the `devlink dpipe table show` entry from `devlink_cmds` below.